hobo.go

```diff
@@ -706,7 +714,7 @@ func runClone(ctx context.Context, cmd *cmdflag.Command, args []string) {
 		// if reuse_home_volume:
 		//   cmd_args += ['--exclude', '*.vmwarevm/home*.vmdk']
 
-		err := runCmd("/usr/bin/tar", "xzvf", archive, "-C", cfg.AppConfig.boxcarsDir())
+		err := runCmd("tar", "xJvf", archive, "-C", cfg.AppConfig.boxcarsDir())
```

Comment:
This is an interesting artifact. OS X tar will actually use the correct decompressor no matter what the contents. This is probably fine.